system-test/install.ts

```diff
@@ -16,36 +16,34 @@
 // ** https://github.com/googleapis/gapic-generator-typescript **
 // ** All changes to this file may be overwritten. **
 
-import {packNTest} from 'pack-n-play';
-import {readFileSync} from 'fs';
-import {describe, it} from 'mocha';
+import { packNTest } from 'pack-n-play';
```

Comment:
Yeah this formatting says it isn't applying `prettier` format rules :)

Reply:
right, double checking. Thanks!

Reply:
This change is back. Thank you!